content/client-server-api/modules/push.md

```diff
@@ -521,8 +521,43 @@ Definition:
 }
 ```
 
+**`.m.rule.is_user_mention`**
+
```

Comment:
```suggestion

{{< added-in v="1.7" >}}

```

Reply:
Likewise in the other rule. And can we add `changed-in` for the deprecated rules?